### ISSUE TITLE
Does not handle insertions correctly

### ISSUE THREAD

**Issue by aeisenberg** [netilon/differify#9]
Here's a simple example that is surprising to me:

```
> d.compare(['a', 'b'], ['a', 'z', 'b'])
{
  _: [
    { original: 'a', current: 'a', status: 'EQUAL', changes: 0 },
    { original: 'b', current: 'z', status: 'MODIFIED', changes: 1 },
    { original: null, current: 'b', status: 'ADDED', changes: 1 }
  ],
  status: 'MODIFIED',
  changes: 2
}
```

I would have expected that the algorithm finds a single `ADDED` change and two `EQUAL` elements because all that has changed between the two arrays is the addition of the second element.

Here is the result I would have expected:

```
> d.compare(['a', 'b'], ['a', 'z', 'b'])
{
  _: [
    { original: 'a', current: 'a', status: 'EQUAL', changes: 0 },
    { original: 'b', current: 'z', status: 'ADDED', changes: 1 },
    { original: null, current: 'b', status: 'EQUAL', changes: 1 }
  ],
  status: 'MODIFIED',
  changes: 1
}
```

Is this a bug or is this by design?

**Comment by fabian-orue** [comment]
Hi aeisenberg , in short, this is not a bug,  it's by design. The current algorithm compares each element in both arrays in the **same order** at the same position (index) and checks if they are equal or there is something that was removed/added. The feature you mention, for the time being is not available.

I will close the issue. Thanks.

**Comment by fabian-orue** [comment]
Hi aeisenberg!, the feature you mentioned was added now in the latest released version **3.2.0**. Please check the README _(Config section)_  :)

**Comment by aeisenberg** [comment]
Thanks! I’ll take a look later.
